typescript-selenium-webdriver-sample/package.json

```diff
@@ -7,6 +7,7 @@
         "@types/jest": "^24.0.23",
         "@types/node": "^12.12.17",
         "@types/selenium-webdriver": "^4.0.5",
+        "accessibility-insights-report": "^0.1.0",
```

Comment:
If we think this has a sufficiently stable interface to be recommending in sample code, should we be giving it a 1.x.y version number? Conversely, if we *don't* think its interface is stable enough to warrant a 1.x.y version number, is it really appropriate for us to be recommending it in sample code?

Reply:
That's a fair point. Let me sync up with the team on if we can call it stable.